### ISSUE TITLE
[PABP] Give reporting.php some love

### ISSUE THREAD

**Issue by daver78** [coa0329/BhaangWeb#197]
Few issues on the page that I can resolve to move us along.

**Comment by daver78** [comment]
@paagla123 You should be able to spot what I updated:
![image](https://user-images.githubusercontent.com/7561473/32292247-bdcc1840-befc-11e7-9461-06b4933e4379.png)

**Comment by daver78** [comment]
@coa0329 Thank you for [E].

@paagla123 Will you have time to read and review questions/notes above before we speak?

**Comment by coa0329** [comment]
@daver78 @paagla123 

1 I hide % if it is (∞%)
2 Fixed bug when user click "today".
3 @daver78 , Can you remove People from the graph ? @paagla123 don't want it.
![049](https://user-images.githubusercontent.com/7914690/32366777-407b5cae-c0b2-11e7-8738-9751da8f7679.JPG)

**Comment by daver78** [comment]
@coa0329 Yes, I'm OK if we remove it from UI.  Or, are you asking me to do the task?

**Comment by coa0329** [comment]
@daver78 
Please tell me how to do it.

**Comment by paagla123** [comment]
@daver78 , @coa0329 not sure how to remove that legend. You can either remove it or email him how to do it. Thx
